featurebyte/models/feature.py

```diff
@@ -78,6 +80,12 @@ class OfflineStoreIngestQueryGraph(FeatureByteBaseModel):
     # output column name of the offline store ingest query graph
     output_column_name: str
     output_dtype: DBVarType
+    # feature job setting of the offline store ingest query graph
+    feature_job_setting: Optional[FeatureJobSetting]
+    # whether the offline store ingest query graph has time-to-live (TTL) component
+    has_ttl: bool
+    # aggregation nodes info of the offline store ingest query graph
+    aggregation_nodes_info: List[AggregationNodeInfo]
```

Comment:
The `AggregationNodeInfo` model has node names in attributes `node_name` and `input_node_names`. To retrieve the nodes corresponding to these node names, is it through the graph attribute above? Checking because there are a few graphs here and there.

Reply:
This is a good catch, we need to map the original node name to ingest graph's node name. Will update this part & add tests to check this functionality.